src/core/__tests__/nested-option.test.tsx

```diff
@@ -312,4 +312,20 @@ describe("nested sub-option", () => {
         expect(Widget.option.mock.calls[0]).toEqual(["option.subOption.d", "def"]);
     });
 
+    describe("special props", () => {
```

Comment:
Consider adding a test that checks for garbage: that the `separetProps` methods removes all special props

Reply:
I mean, adding instead of this test